Dwolla.Client/Models/Responses/FundingSource.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Dwolla.Client.Models.Responses
+{
+    public class FundingSource : BaseResponse
+    {
+        public string Id { get; set; }
+        public string Status { get; set; }
+        public string Type { get; set; }
+        public string BankAccountType { get; set; }
+        public string Name { get; set; }
+        public DateTime Created { get; set; }
+        public bool Removed { get; set; }
+        public string[] Channels { get; set; }
```

Comment:
Why not use `List<string>` (same for the other arrays)?

Reply:
`List<string>` implies `Add()` and similar methods in my mind. I can switch to `List<>` if that's what you'd prefer.